platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/admin/core/api/ConfigurationAdmin.java

```diff
@@ -80,6 +81,13 @@
   List<Service> listServices(String serviceFactoryFilter, String serviceFilter);
 
   /**
+   * Returns all configurations that satisfy the provided filter.
+   *
+   * @see org.osgi.service.cm.ConfigurationAdmin#listConfigurations(String)
+   */
+  Configuration[] listConfigurations(String filter) throws IOException, InvalidSyntaxException;
```

Comment:
❓ Why don't we go directly to osgi 's config admin for this?

Reply:
Because I need DDF's config admin for the object class definition lookup and they have the same interface name. I didn't want to deal with fully qualified package naming.

Reply:
I don't think we should be modifying the API for this. We are now adding something that wasn't in the API just to avoid fully package names. The fact is that we want to move away from that API at some point in favor of moving some of that logic down below.

Reply:
The good news is that this API is not whitelisted. So subsequent changes / migrations **can** be done.

Reply:
The problem is that by introducing it, others might start using it and making it more difficult to undo later.

Reply:
Based on our discussion, your follow up PR will undo this right?